### PR TITLE
Topic/351 make checkboxes accessible

### LINKED ISSUE #351: make checkboxes accessible
Currently, checkboxes are not part of the tab-flow because it's set to display none.
We should work on a solution that allowed them to be a part of the tab- flow

### PR DESCRIPTION
**Pull request by frontendherodk**
Fixes #351 

You're now able to tap through checkboxes and radio buttons.

### CODE REVIEW

File: src/styles/atoms/_radio.scss [hunk 1]
```diff
@@ -44,6 +47,10 @@
         }
     }
 
+    &:focus + #{$sibling-label-selector}:before {
+        border-color: #26aafe;
```
Comment (rbarilani) [inline src/styles/atoms/_radio.scss]: @frontendherodk Does it make sense to have a variable for this value?

Comment (frontendherodk) [inline src/styles/atoms/_radio.scss]: Ah yes. My bad. Let me quickly fix that

**Comment by rbarilani** [comment]
👍

**Comment by frontendherodk** [comment]
👍

**Pull request merged**